kibana/kibana.yml

```diff
@@ -19,8 +19,8 @@ kibana_index: ".kibana"
 # used by the Kibana server to perform maintence on the kibana_index at statup. Your Kibana
 # users will still need to authenticate with Elasticsearch (which is proxied thorugh
 # the Kibana server)
-kibana_elasticsearch_username: kibana
-kibana_elasticsearch_password: changeme
+#kibana_elasticsearch_username: kibana
```

Comment:
Uncomment out these two lines please, at least until we change how we do auth throughout the logging stack